features/customer_booking_request.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Customer creates a Booking Request
+  As a customer
+  I want to request an appointment
+  So that I can meet for guidance
```

Comment:
@benbarnett Changed to `So I can understand my pension options` which reads nicer too.